manifest.json

```diff
@@ -125,6 +125,7 @@
         "language_marte": {},
         "language_meson": {},
         "language_miniscript": {},
+        "language_monkeyc": {},
```

Comment:
You should also bump the version of `meta_languages`.

Reply:
meta_languages in the manifest itself, yeah?

Reply:
Yep.